lib/web_console/middleware.rb

```diff
@@ -58,6 +59,25 @@ def call(env)
 
     private
 
+      # Returns whether the response is from an acceptable content type.
+      #
+      # We can render a console for HTML and TEXT by default.
+      def acceptable_response_content_type?(headers, env)
```

Comment:
Did you moved that out of the request object, because you needed the Rack headers? If its that, I think we can keep the method in the request object by making it take the rack headers as input.